eng/Version.Details.xml

```diff
@@ -66,25 +66,25 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>f6192d1e284a08ac05041d05fa6e60dec74b24f5</Sha>
     </Dependency>
-    <Dependency Name="optimization.windows_nt-x64.IBC.CoreFx" Version="99.99.99-master-20190716.1">
+    <Dependency Name="optimization.windows_nt-x64.IBC.CoreFx" Version="99.99.99-master-20200806.6">
```

Comment:
so this implies these havent been updated for more than a year?

Reply:
These were last updated with data from February 28, 2020.  You can see that in the change to version.props.  Version.Details.xml is used to keep track of changes that come in via dependency flow.  I just didn't update Version.Details.xml last time around.

Reply:
ok, thanks for the info.